source/configuration_manual/authentication/oauth2.rst

```diff
@@ -98,41 +113,32 @@ passdb settings
 
 .. code-block:: none
 
-  auth_oauth2_config_file = /usr/local/etc/dovecot/dovecot-oauth2.token.conf.ext
-
   passdb oauth2 {
-    mechanisms = plain login
-    args = /usr/local/etc/dovecot/dovecot-oauth2.plain.conf.ext
+    oauth2 {
+      # common settings can be inherited from global scope
+      mechanisms = plain login
```

Comment:
I don't think this works inside oauth2 { .. }, since it's passdb_mechanisms setting that is used by passdb lookup to filter which mechanisms to use. There's no oauth2 at that point.